CHANGELOG.md

```diff
@@ -6,9 +6,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Support for using `Deferred` secrets fetching via `conjur::secret`.
+  [cyberark/conjur-puppet#13](https://github.com/cyberark/conjur-puppet/issues/13).
+
+### Changed
+- `conjur::secret` now must be used as a `Deferred` function. Method signature has
+  changed as well. [cyberark/conjur-puppet#13](https://github.com/cyberark/conjur-puppet/issues/13).
```

Comment:
Does the issue talk more about the method signature change? It might be nice to add something (even if short) there, because this bread crumb trail might lead people there looking for that info.

Reply:
Added: https://github.com/cyberark/conjur-puppet/issues/13#issuecomment-672209234